library/restoreSession.php

```diff
@@ -6,7 +6,7 @@
 //
 var oemr_session_name = '<?php echo session_name(); ?>';
 var oemr_session_id   = '<?php echo session_id(); ?>';
-var oemr_dialog_close_msg = '<?php echo xl('OK to close this other popup window?'); ?>';
+var oemr_dialog_close_msg = '<?php echo "OK to close this other popup window?"; ?>';
```

Comment:
TODO for me:
Use the xla() function if it exists via function_exists('xla') call